src/style.css

```diff
@@ -1 +1,74 @@
-/* Styles go here */
+* {
+  color: #000;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  text-decoration: none;
+}
+
+.link:hover,
+.is-active {
+  color: #00acdc;
+}
+
+header {
+  background: #fff;
+  display: flex;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+}
+
+.logo {
+  display: flex;
+  width: 40px;
+  height: 40px;
+  padding: 10px 0;
+}
+
+.nav {
+  display: flex;
+  background: #fff;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: space-between;
+  height: 100%;
+  width: 100%;
+}
+
+.nav__item {
+  display: flex;
+  align-items: center;
+  margin-left: 20px;
+  font-style: normal;
+  font-weight: 500;
+  text-transform: uppercase;
+  height: 100%;
+  color: #000;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.item__link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 100%;
+}
+
+.is-active:after {
+  content: "";
+  width: 100%;
+  height: 4px;
+  border-radius: 8px;
+  background: #00acdc;
+  position: absolute;
+  bottom: 0;
+  left: 0;
```

Comment:
Please pay attention to your CSS properties order
https://9elements.com/css-rule-order/